caffe2/utils/math_cpu.cc

```diff
@@ -784,6 +785,16 @@ DELEGATE_CBRT_FUNCTION(float)
 DELEGATE_CBRT_FUNCTION(double)
 #undef DELEGATE_CBRT_FUNCTION
 
+#define DELEGATE_ERF_FUNCTION(T)                                    \
+    template <>                                                     \
+  C10_EXPORT void Erf<T, CPUContext>(                               \
+      const int N, const T* X, T* Y, CPUContext*) {                 \
+    std::transform(X, X + N, Y, [](const T x) { return erf(x); });  \
```

Comment:
Though in eigen's doc, erf function is not using SIMD, I'd suggest to use eigen here. It may update in their future version.
https://eigen.tuxfamily.org/dox/group__CoeffwiseMathFunctions.html

```suggestion
    EigenVectorArrayMap<T>(Y, N) = ConstEigenVectorArrayMap<T>(X, N).erf();  \
```

Reply:
@BIT-silence the problem of using eigen here is erf is in unsupported, and android doesn't build

Reply:
Interesting, I'm not sure why erf is in unsupported part.